docs/interfaces/modem/modem.md

```diff
@@ -96,5 +98,14 @@ error:
 +CME ERROR: SIM not inserted
 ```
 
+## Global Positioning System (GPS)
+
+### Time
+
+The GPS time can be extracted from the `$GPRMC` string outputted by the GPS.
```

Comment:
maybe add that gps-parser is possible to use in most of our system or can be download using opkg.

Reply:
Do you mean [gpsd](https://gpsd.io/)?

Reply:
no but looks like we have gpsd on HMX but not gps-parser (i tried to find it in opkg list and maybe it is an older tool). 
gps2udp       gpscat        gpscsv        gpsctl        gpsd          gpsdctl       gpsdebuginfo  gpsdecode     gpsfake       gpsmon        gpspipe       gpsplot       gpsprof       gpsrinex      gpssnmp       gpssubframe

Reply:
it is okay